src/main/java/com/wolox/training/repositories/BookRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.wolox.training.repositories;
+
+import com.wolox.training.models.Book;
+import org.springframework.data.repository.Repository;
+
+public interface BookRepository extends Repository<Book, Long> {
+
+    Book findByAuthor(String author);
```

Comment:
Wrap the book entity in Optional, this has a lot of advantage. Check this https://www.baeldung.com/java-optional

Reply:
excellent! very useful type